contracts/utils/AccessControl.sol

```diff
@@ -41,7 +41,7 @@ contract AccessControl {
    * @dev Has all the bits are enabled (2^256 - 1 value)
    */
   // solhint-disable-next-line
-  uint256 private constant FULL_PRIVILEGES_MASK = type(uint256).max; // before 0.8.0: uint256(-1) overflows to 0xFFFF...
+  uint256 public constant FULL_PRIVILEGES_MASK = type(uint256).max; // before 0.8.0: uint256(-1) overflows to 0xFFFF...
```

Comment:
I think you just need to make it `internal`.

Reply:
i want it to be accessible from outside.

Reply:
But that is a constant. You know it
It is `type(uint256).max` >> `0xfffffffffff....`